specification/cognitiveservices/data-plane/Speech/TextToSpeech/preview/2023-12-01-preview/texttospeech.json

```diff
@@ -281,6 +281,9 @@
         ],
         "description": "Deletes the consent identified by the given ID.",
         "operationId": "Consents_Delete",
+        "produces": [
+          "application/json"
```

Comment:
This is not breaking change. Service return error response in json from the beginning.
Add this to fix LintDiff error.